docs/user_guide/scripts/02_medrecord_intro.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+
+import medmodels as mm
+
+# Patients DataFrame (Nodes)
+patients = pd.DataFrame(
+    [
+        ["Patient 01", 72, "M", "USA"],
+        ["Patient 02", 74, "M", "USA"],
+        ["Patient 03", 64, "F", "GER"],
+    ],
+    columns=["ID", "Age", "Sex", "Loc"],
+)
+
+# Medications DataFrame (Nodes)
+medications = pd.DataFrame(
+    [["Med 01", "Insulin"], ["Med 02", "Warfarin"]], columns=["ID", "Name"]
+)
+
+# Patients-Medication Relation (Edges)
+patient_medication = pd.DataFrame(
+    [
+        ["Patient 02", "Med 01", pd.Timestamp("20200607")],
+        ["Patient 02", "Med 02", pd.Timestamp("20180202")],
+        ["Patient 03", "Med 02", pd.Timestamp("20190302")],
+    ],
+    columns=["Pat_ID", "Med_ID", "Date"],
+)
+
+record = mm.MedRecord.builder().add_nodes((patients, "ID"), group="Patients").build()
+
+# TODO: Adding new nodes to group must be added as issue in next MedModels weekly
+record.add_nodes((medications, "ID"))
```

Comment:
should be
```suggestion
record.add_nodes((medications, "ID"), "Medications")
```
once #203 is merged which will be in a few minutes probably.

Also remove the TODO above and the `add_group` below